superscore/control_layers/core.py

```diff
@@ -98,7 +99,11 @@ def get(self, address: Union[str, Iterable[str]]) -> Union[EpicsData, Iterable[E
     @get.register
     def _get_single(self, address: str) -> EpicsData:
         """Synchronously get a single ``address``"""
-        return asyncio.run(self._get_one(address))
+        try:
+            res = asyncio.run(self._get_one(address))
+        except CommunicationError as e:
+            res = e
+        return res
```

Comment:
One more silly nitpick, no need to do any variable assignments here
```suggestion
        try:
            return asyncio.run(self._get_one(address))
        except CommunicationError as e:
            return e
```